fixtures/chauvet-dj/intimidator-spot-110.json

```diff
@@ -38,25 +38,39 @@
           "type": "Open"
         },
         {
-          "type": "Color"
+          "type": "Color",
+          "name": "Color 1 (Red)",
```

Comment:
Can you please rename these wheel slots, so they are only called "Red", "Green", …?